File: hard/17_detect_arbitage/detect_arbitage.test.js
// import { underscorifySubstring } from './underscorify_substring.js';
const detectArbitrage = require('./detect_arbitage.js');

describe('TestProgram', () => {
  it('Test Case 1', () => {
    const input = [[1.0, 0.8631, 0.5903], [1.1586, 1.0, 0.6849], [1.6939, 1.46, 1.0]];
    const expected = true;
    const actual = detectArbitrage(input);
    expect(actual).toEqual(expected);
  });
});

File: hard/17_detect_arbitage/detect_arbitage_test.py
import unittest
from detect_arbitage import detectArbitrage

class TestProgram(unittest.TestCase):
    def test_case_1(self):
        input = [[1.0, 0.8631, 0.5903], [1.1586, 1.0, 0.6849], [1.6939, 1.46, 1.0]]
        expected = True
        actual = detectArbitrage(input)
        self.assertEqual(actual, expected)
